arch/lkl/kernel/init.c

```diff
@@ -6,6 +6,25 @@ int lkl_init(struct lkl_host_operations *ops)
 {
 	lkl_ops = ops;
```

Comment:
Please put this after the config checks. I don't think `lkl_ops` should be initialized on failure...

Reply:
Done, thanks !